src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -145,7 +145,8 @@ export class EditorPart extends Part implements IEditorPart, IEditorGroupService
 				previewEditors: editorConfig.enablePreview,
 				showIcons: editorConfig.showIcons,
 				showTabs: editorConfig.showTabs,
-				showTabCloseButton: editorConfig.showTabCloseButton
+				showTabCloseButton: editorConfig.showTabCloseButton,
```

Comment:
Just delete support for `showTabCloseButton` option since it is now supported via your `tabCloseButton` setting.